<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdgeDetection</class>
 <widget class="QMainWindow" name="EdgeDetection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HighEdit">
    <property name="geometry">
     <rect>
      <x>672</x>
      <y>110</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="HighSlider">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>高阈值：</string>
    </property>
   </widget>
   <widget class="QSlider" name="LowSlider">
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>140</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>598</x>
      <y>170</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>低阈值:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LowEdit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="HoughCircleBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>260</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>霍夫圆变化</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HoughEdit">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>330</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>阈值：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="HoughLineBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>霍夫线变化</string>
    </property>
   </widget>
   <widget class="QSlider" name="HoughSlider">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>HighEdit</zorder>
   <zorder>img</zorder>
   <zorder>SaveButton</zorder>
   <zorder>HighSlider</zorder>
   <zorder>label_2</zorder>
   <zorder>LowSlider</zorder>
   <zorder>label_3</zorder>
   <zorder>LowEdit</zorder>
   <zorder>HoughCircleBox</zorder>
   <zorder>HoughEdit</zorder>
   <zorder>label_4</zorder>
   <zorder>HoughLineBox</zorder>
   <zorder>HoughSlider</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
